app/lib/google_cloud_compute/vms_wrapper.rb

```diff
@@ -0,0 +1,17 @@
+module GoogleCloudCompute
+  class VmsWrapper
```

Comment:
Maybe a better name? :thinking: 
```suggestion
  class ComputeCollection
```